### PR TITLE
fix: blocked page

### LINKED ISSUE #866: Auth flow issue using Discord (maybe others)
If you protect your map with discord authentication flow and restrict the map to be only available for users in [allowed guild], when a user logins and isn't part of allowed guilds, the Map hangs showing the loading animation. On the address bar one can see https://serverurl/blocked/message%3Daccess_denied but the user is never clearly informed that his auth failed in some way. There should be a landing page option to where the user is redirected in case he's denied access to inform him of whatever the map owner wishes.

### PR DESCRIPTION
**Pull request by TurtIeSocks**
Resolves #866 

- Makes sure to fetch server settings before rendering blocked page
- Also makes sure to hide the ghetto html loading el

### CODE REVIEW

**Comment by tchavei** [comment]
I'm testing

Right now the access denied page is perfect. See picture 1. Can we personalize it if needed?

image


There's only one issue. When initially going to the start page, the custom login is now at the bottom (hidden) and all one can see is "loading". See picture 2

image


Thank you

**Comment by TurtIeSocks** [comment]
Oops, fixed.

**Comment by tchavei** [comment]
Works!

A little strange looking buttons on mobile but I can live with that. Great job! Kudos 

Screenshot_2023-11-02-23-47-53-12_40deb401b9ffe8e1df2f1cc5ba480b12

File: src/components/layout/auth/Blocked.jsx [hunk 1]
```diff
@@ -12,79 +13,61 @@ import DiscordLogin from './Discord'
...
       justifyContent="center"
       alignItems="center"
-      style={{ minHeight: '95vh' }}
+      height="100cqh"
+      py={10}
     >
-      <Grid item>
+      <Grid xs={11}>
         <Typography variant="h3" align="center">
           {t('access')} {t('denied')}!
```
Comment (kamieniarz) [inline src/components/layout/auth/Blocked.jsx:32]: Could we actaully have `{t('no_access')} {t('denied')}!` here?

**Pull request merged**